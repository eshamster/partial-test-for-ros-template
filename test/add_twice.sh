#!/bin/bash

set -eu

test_name=test-duplicated-add
file=sample

function ros_t()
{
    ros template $@
}

ros_t deinit ${test_name} > /dev/null 2>&1 && :
ros_t init ${test_name}
ros_t checkout ${test_name}
ros_t rm ${file}

echo "First" > ${file}
ros_t add ${file}

ros_t list
ros_t cat ${file}
ros_t type djula ${file}

echo "Second" > ${file}
ros_t add ${file}

ros_t list
ros_t cat ${file}
