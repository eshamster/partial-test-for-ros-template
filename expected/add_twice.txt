      copy  sample 
First
      djula sample 
Second
